src/AlgebraicGeometry/Schemes/CoveredSchemes/Objects/Properties.jl

```diff
@@ -1,29 +1,122 @@
-export is_empty
+export is_empty, is_smooth, is_reduced, is_integral, is_irreducible
 
 ########################################################################
 # Properties of AbsCoveredSchemes                                      #
+#    by referral to the patches                                        #
 ########################################################################
+
+########################################################################
+# (1) Check and store, whether a covered scheme is empty               #
+########################################################################
+@doc Markdown.doc"""
+   is_empty(X::AbsCoveredScheme)
+
+Return the boolean value whether a covered scheme `X` is empty.
+
+"""
 @attr function is_empty(X::AbsCoveredScheme)
   if !isdefined(X, :coverings) 
     return true
   end
   return all(x->isempty(x), all_patches(default_covering(X)))
```

Comment:
`all(isempty, all_patches...)`
Isempty is already a function

Reply:
Ah, that explanation helps. But there are so many of these in the file (and many not even touched by this PR) that shall do these changes in a separate PR, which does not introduce new functionality.